resources/test/tests/idlharness/IdlInterface/test_to_json_operation.html

```diff
@@ -32,18 +32,14 @@
     i = interfaceFrom("interface B { [Default] object toJSON(); attribute long foo; };");
     i.test_to_json_operation(wrap({ foo: "a value" }), i.members[0]);
 
-    // should fail (extra property)
+    // should handle extra-members (e.g. fron an extension specification)
```

Comment:
nit: should this be "attributes" instead of "members"?

Reply:
Yes. Also fron -> from.